### PR TITLE
RE-468 add branchId and branchSlug to the plugins sdk

### PR DESCRIPTION
**Pull request by ugeshmalempat**
Recently we have added branchId and branchSlug to sso payload plugins(https://github.com/Staffbase/backend/pull/330). This PR will make sure that the plugin sdk will also have the branchId and slug.

### CODE REVIEW

File: src/main/java/com/staffbase/plugins/sdk/sso/SSOData.java [hunk 1]
```diff
@@ -109,23 +109,27 @@
...
   /**
    * The key in the JWT claims for fetching the list of tags
-   * that are configured in the Staffbase app. 
+   * that are configured in the Staffbase app.
    */
   public static final String KEY_TAGS = "tags";
 
+  public static final String KEY_BRANCH_ID = "branch_id";
+
+  public static final String KEY_BRANCH_SLUG = "branch_slug";
```
Comment (scthi) [inline src/main/java/com/staffbase/plugins/sdk/sso/SSOData.java]: Please respect alphabetical order here and below.

File: src/main/java/com/staffbase/plugins/sdk/sso/SSOData.java [hunk 2]
```diff
@@ -439,10 +449,20 @@ public boolean isDeleteInstanceCall() {
...
+
+  public String getBranchSlug() {
+    return this.branchSlug;
+  }
+
   @Override
   public String toString() {
     return "SSOData ["+
   " instanceID="+ this.instanceID+
+  ", branchId="+ this.branchId+
```
Comment (scthi) [inline src/main/java/com/staffbase/plugins/sdk/sso/SSOData.java]: The SDK currently uses ID instead of Id. For consistency reasons, I'd recommend to use `branchID`.

Switching everything to `Id` should be a separate PR.

File: src/main/java/com/staffbase/plugins/sdk/sso/SSOData.java [hunk 3]
```diff
@@ -131,72 +135,76 @@
...
 
   /**
-   * The unique id of the specific plugin instance that was requested using
-   * staffbase's SSO.
+   * The name of the audience the staffbase's SSO data is dedicated.
    */
-  private final String instanceID;
+  private final String audience;
+
+  private final String branchID;
```
Comment (Drako) [inline src/main/java/com/staffbase/plugins/sdk/sso/SSOData.java]: please add documentation comments to `branchID` and `branchSlug` just like the other private fields have them.
